src/cli/build.rs

```diff
@@ -2,6 +2,29 @@ use std::fs;
 use std::process::Command;
 use rustc_version::version_meta;
 
+#[cfg(target_os = "macos")]
```

Comment:
code seems to be duplicate for macOS and non Mac unix.  only difference is argument types.  so make argument var on OS